src/Console/SupervisorCommand.php

```diff
@@ -74,6 +75,10 @@ public function handle(SupervisorFactory $factory)
      */
     protected function start($supervisor)
     {
+        if ($supervisor->options->nice > 0) {
```

Comment:
- `0` is a valid niceness level
- so are negative values

This check, see my other comment, should be `!== null`

It doesn't matter that e.g. negative values are only allowed for root user rights. Don't outsmart the dev who might have run as root _for whatever_ reason, etc.